TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -289,6 +289,13 @@ public static void OnCodeElement(StartStatement statement, CodeElementsParser.St
         }
     }
 
-    #endregion
+    class AlterStatementChecker
+    {
+        public static void OnCodeElement(AlterStatement statement, CodeElementsParser.AlterStatementContext context)
+        {
+            DiagnosticUtils.AddError(statement, "ALTER should not be used", context, MessageCode.Warning);
```

Comment:
You don't need to put the variable "context" here.
The error message will be enough for the user to understand the diagnostic.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/commit/97398cf937bc4578f4bde331f46cf26fd3c9801d